src/test/resources/discovery/specification-sample-1.html

```diff
@@ -11,17 +11,20 @@ <h1>Contents</h1>
     </ul>
 </section>
 <section about="https://example.org/specification1#spec1" typeof="spec:NormativeRequirement">
-    <h1>Requirement 1</h1>
+    <h1 property="schema:name">Requirement 1</h1>
+    <p property="schema:description">Full <span property="spec:excerpt">text of requirement 1</span></p>
```

Comment:
Include `datatype="rdf:HTML"` on nodes with properties with literal range when the markup should be preserved (allows processing to continue).